dashboard/common/utils.py

```diff
@@ -1,6 +1,14 @@
-import logging, os
+import logging, os, json
 logger = logging.getLogger(__name__)
 
+try:
```

Comment:
ENV is already available as settings.ENV, just 

`from django.conf import settings
`
and use settings.ENV. You don't have to re-read this file here.